tests/saw/sike/spec/include/keccak.cry

```diff
@@ -0,0 +1,238 @@
+/*
+ * Copyright (c) 2013 David Lazar <lazard@galois.com>
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
```

Comment:
Can this be updated to apache 2.0 to match the rest of the code? Or update tests/saw/sike/LICENSE.TXT with a subcomponent section with the MIT license listed explicitly for this file like https://github.com/awslabs/s2n/blob/master/LICENSE#L206